.github/workflows/build-wheels.yml

```diff
@@ -120,11 +120,13 @@ jobs:
           - {os: windows-2019, arch: arm64}
           # https://github.com/pytorch/pytorch/issues/125109
           - {os: windows-2019, torch-version: '2.3'}
+          - {os: windows-2019, torch-version: '2.4'}
           # arm64-macos is only supported for torch >= 2.0
           - {os: macos-14, arch: arm64, torch-version: '1.12'}
           - {os: macos-14, arch: arm64, torch-version: '1.13'}
           # x86_64-macos is only supported for torch <2.3
           - {os: macos-13, arch: x86_64, torch-version: '2.3'}
+          - {os: macos-13, arch: x86_64, torch-version: '2.4'}
```

Comment:
Is the origin of excluding macos the same as the issue for windows which is linked above?

Reply:
no, it's that PyTorch no longer distribute Intel macOS builds (there is a comment just above these two lines, but I can try to make it clearer!)

Reply:
It is fine but maybe bit clearer could be something like

`# arch x86_64 on macos is only supported for torch <2.3`